matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/sync/ReadReceiptHandler.kt

```diff
@@ -110,4 +122,13 @@ internal class ReadReceiptHandler @Inject constructor() {
             }
         }
     }
+
+    fun getContentFromInitSync(roomId: String): ReadReceiptContent? {
+        @Suppress("UNCHECKED_CAST")
+        return roomSyncEphemeralTemporaryStore.read(roomId)
+                ?.also { roomSyncEphemeralTemporaryStore.delete(roomId) }
```

Comment:
If something fails during the sync transaction we will loose the initial read receipts

Reply:
Yes, good point, I'm not sure how to handle that... Will try to find a solution